src/System.Private.ServiceModel/src/System/ServiceModel/Channels/ServiceModelHttpMessageHandler.CoreClr.cs

```diff
@@ -10,10 +10,10 @@
 
 namespace System.ServiceModel.Channels
 {
-#if FEATURE_CORECLR && !TARGETS_WINDOWS
+#if FEATURE_CORECLR
     public partial class ServiceModelHttpMessageHandler
     {
-        private HttpClientHandler _innerHandler;
+        private readonly HttpClientHandler _innerHandler;
```

Comment:
For future, consider making the same "readonly" change to the NetNative HttpMessageHandler.